src/universal/components/SignInPage/Header.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * The brand header for the authentication homepages.
+ *
+ * @flow
+ */
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+import appTheme from 'universal/styles/theme/appTheme';
+import parabolLogo from 'universal/styles/theme/images/brand/parabol-beta-lockup.svg';
+
+
+const headerStyles = {
```

Comment:
I used basic inline styles. Happy to change them to use emotion or aphrodite, but this was easiest for development.